src/common/utils.js

```diff
@@ -1 +1,31 @@
-// TODO Menu creator object
+import React from 'react'
+import uuid from 'uuid'
+import { Icon, Menu } from 'antd'
+import { Link } from 'react-router-dom'
+
+const { Item, SubMenu } = Menu
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Don't use `// eslint-disable-next-line import/prefer-default-export`  the meaning is lost in eslint

Reply:
It's in utils file. There is only one function now and because that i get error 'prefer default export'. In future I assume we will have more than one export